src/mempools/global_mempool.hpp

```diff
@@ -50,7 +50,7 @@ namespace argo {
 				 */
 				global_memory_pool(std::size_t size) {
 					size+=reserved;
-					backend::init(size);
+					backend::init(size, 1<<30); // 1GB of memory for cache as default (= 1<<30 bytes)
```

Comment:
The default value of 1GB already slows down the tests significantly.
Using a larger value, like 8GB causes frequent test timeouts even on a single node.
This may or may not be linked to the fact that the tests do not allocate that much memory, thus the cache would be larger than the distributed memory area.

Reply:
fixed in 1a3dc8a86acbf21bdbc3650c8230326a4d87ca7d and #38